internal/auth/bypass.go

```diff
@@ -32,8 +32,13 @@ func Bypass(data []byte) (bool, error) {
 	if err := json.Unmarshal(data, &baseMessage); err != nil {
 		return false, fmt.Errorf("parse error: %w", err)
 	}
-	if !slices.Contains(methodsWhitelist, baseMessage.Method) {
+	if !BypassMethod(baseMessage.Method) {
```

Comment:
What's the value of this change?

Reply:
TL;DR - For now, none 😄  for the future, maybe.

I wanted to access `methodsWhitelist` without exposing the slice (keep it read-only) to check if a method requires authentication or not. And as I had to add this `BypassMethod`, I thought it would be wiser to call it from the `Bypass` function in case we add more logic to it in the future.